### DISCUSSION TITLE
Turn off pyrigth

### DISCUSSION THREAD

**Discussion by alextk37** [LazyVim/LazyVim#4082] [Q&A] [answered]
Hi everybody. Guys, tell me how to turn off the automatic installation of pyright when opening a python file.
Or tell me how to solve the problem with imports to Django in pyright, which are not present in ruff_lsp.

**Comment by dpetka2001** [comment, marked as answer]
See #3907

**Reply by alextk37** [reply]
Thanks 🙏
